### PR TITLE
Feature/tabs menu

### PR DESCRIPTION
**Pull request by mluena**
- Styles in impact sidebar

### CODE REVIEW

File: app/components/elements/AreaSummary/style.scss [hunk 1]
```diff
@@ -3,18 +3,24 @@
...
+    > span {
+      margin: auto 0;
+    }
   }
+
   .area-summary-wrapper {
     display: flex;
     align-items: baseline;
+    &.impact {
+      display: flex;
```
Comment (agnlez) [inline app/components/elements/AreaSummary/style.scss:15]: Not necessary. It already inherits it from `area-summary-wrapper`.

File: app/components/areas/Sidebar/Reach.jsx [hunk 2]
```diff
@@ -49,7 +49,7 @@ class ReachSidebarArea extends React.Component {
     const maxValue = Math.max.apply(null, programs.map((p) => {
```
Comment (agnlez) [inline app/components/areas/Sidebar/Reach.jsx:49]: Move it to a selector.

Comment (mluena) [reply app/components/areas/Sidebar/Reach.jsx:49]: there is no redux in this project

File: app/components/elements/AreaSummary/index.jsx [hunk 3]
```diff
@@ -61,6 +60,9 @@ class AreaSummary extends React.Component {
               />
             </div>}
         </div>
+        {program !== "overall" && (<span className="subtitle">
+            {programs.find((p) => p.id === program).label}
```
Comment (mbarrenechea) [inline app/components/elements/AreaSummary/index.jsx:68]: What would happen if we don't find anything? I can tell you, ERROR

File: app/components/elements/AreaSummary/index.jsx [hunk 4]
```diff
@@ -37,11 +37,10 @@ class AreaSummary extends React.Component {
       <div className={classname(
         location.href.includes("reach") ? 'reach' : 'impact',
         'c-area-summary')}>
-        <div className="area-summary-wrapper">
+        <div className={classname(
```
Comment (mbarrenechea) [inline app/components/elements/AreaSummary/index.jsx]:
```suggestion
        <div className={classname({
          'area-summary-wrapper': true,
          'impact': !location.href.includes("reach"),
          'reach': location.href.includes("reach")
        })
```

**Review by mbarrenechea — CHANGES_REQUESTED** [review]

**Review by mbarrenechea — APPROVED** [review]

**Pull request merged**